### DISCUSSION TITLE
Accessing login page

### DISCUSSION THREAD

**Discussion by bakytk** [docker-mailserver/docker-mailserver#2898] [Q&A] [answered]
Hello, I'm pretty new to backend, but want to try this container for personal use. Not able to login-page, do I need to add this functionality myself?

 As far as I understand, process is roughyl as well:

1. Set DNS config (Subdomain name) to render login page: eg "mail.example.com" --> my.ip.server.number
2. NGINX.conf file should have "/mail" route or sth to forward to Docker_container: PORT (eg 8088)
3. There's UI component (pre-built I guess) running at  Docker_container: PORT to give back login page

Pls someone clarify, stuck for 3-4 days- not sure how to check login page accessibility externally

**Comment by casperklein** [comment, marked as answer]
> Not able to login-page, do I need to add this functionality myself? As far as I understand, process is roughyl as well:

Why do you assume, that there is a web gui? There is none.
